site/src/Librecores/ProjectRepoBundle/Entity/ProjectRepository.php

```diff
@@ -58,4 +58,23 @@ public function findByFqnameFragment($fqnameFragment, $limit = null)
         }
         return $p->getResult();
     }
+
+    /**
+     * Find all projects ordered by recent activity (last modified)
+     *
+     * @param int $limit
+     * @return array of projects
+     */
+    public function findByRecentActivity($limit = null)
+    {
+		$p = $this->getEntityManager()
+			->createQuery(
+				'SELECT p '.
+				'FROM LibrecoresProjectRepoBundle:Project p '.
+				'ORDER BY p.dateLastModified DESC');
+		if ($limit != 0) {
+			$p->setFirstResult(0)->setMaxResults($limit);
+		}
+		return $p->getResult();
+    }
 }
```

Comment:
please indent by four spaces. If you use Eclipse with the PDT/Symfony plugin (highly recommended), switch to the PSR-2 coding style (more or less common across PHP projects these days). I think Netbeans also has this style prepackaged.